go/libkb/env.go

```diff
@@ -541,14 +543,28 @@ func (e *Env) GetServerURI() string {
 	// check for test flag here in order for production api endpoint
 	// tests to pass.
 	if e.Test.UseProductionRunMode {
-		return ServerLookup[e.GetRunMode()]
+		server, e := ServerLookup(e, e.GetRunMode())
+		if e != nil {
+			// ServerLookup only returns an error if the RunMode is bogus. Panic since there is no
+			// way to get a URL in this case
+			panic(e)
+		}
+		return server
 	}
 
 	return e.GetString(
 		func() string { return e.cmd.GetServerURI() },
 		func() string { return os.Getenv("KEYBASE_SERVER_URI") },
 		func() string { return e.GetConfig().GetServerURI() },
-		func() string { return ServerLookup[e.GetRunMode()] },
+		func() string {
+			server, e := ServerLookup(e, e.GetRunMode())
```

Comment:
Call your error `err` since that's the convention

Reply:
Done